source/overview.rst

```diff
@@ -0,0 +1,439 @@
+behind them, as well as their dependencies.
+This includes only repositories that are hosted on pcdshub with a
+corresponding online documentation page.
+
+Hutch-specific Repositories
+---------------------------
+============= ================================================================
+Repository    `amo <https://github.com/pcdshub/amo>`_,
+              `sxr <https://github.com/pcdshub/sxr>`_,
+              `xpp <https://github.com/pcdshub/xpp>`_,
+              `xcs <https://github.com/pcdshub/xcs>`_,
+              `mfx <https://github.com/pcdshub/mfx>`_,
+              `cxi <https://github.com/pcdshub/cxi>`_,
+              and `mec <https://github.com/pcdshub/mec>`_
+
+How we use it We put softlinks like ``mfx3`` in the hutch opr's path. These
+              link to scripts like ``mfxpython`` in the repository that launch
+              a convenient ``IPython`` environment for controlling the hutch
+              and automating DAQ data collection. The repository is used to
+              host hutch-specific classes, devices, and configurations that
+              may later be merged into a shared module.
+
+Motivation    We need dedicated landing areas for hutch-specific and
+              experiment-specific code to satisfy the needs of a constantly
+              shifting environment. This can't go into a module because the
+              module can't control which python environment you launch it
+              from.
+
+How it works  A version file e.g. ``mfxversion`` chooses a shared conda
+              environment and may set up development package overrides. The
+              ``mfxpython`` script enters this environment using
+              :ref:`hutch-python` and a brief ``conf.yml`` configuration file.
+              These coordinate the loading of shared services and provide
+              hooks for the hutch-specific code.
+
+Dependencies  This may vary between hutches, but all hutches rely on
+              :ref:`hutch-python` as the launcher and configuration
+              reader. All of these repositories were created from the
+              ``hutch-python --create reponame`` template function.
+============= ================================================================
+
+hutch-python
+------------
+============= ================================================================
+Repository    `hutch-python <https://github.com/pcdshub/hutch-python>`_
+
+How we use it Scripts like ``mfxpython`` call on the ``hutch-python``
+              application with a hutch-specific ``conf.yaml`` file, as well as
+              hutch-specific ``mfx/beamline.py`` and ``experiments`` folder.
+
+Motivation    The load process for each hutch is essentially identical. This
+              should be handled in shared code with simple hooks for
+              hutch-specific and experiment-specific additions. The best
+              alternaive is using ``IPython`` profiles which can get messy and
```

Comment:
`alternative`

Reply:
I would explain this slightly differently. 
```
"This was chosen instead of supporting IPython profiles or massive `beamline.py` files which ..."
```

Reply:
alternaive: adjective. Being of a different kind of naive

Reply:
Good catches, I'll make the edit. I figure this doc has a lot of places where it can be fixed or refined.